js/data/languages.js

```diff
@@ -0,0 +1,121 @@
+const languages = [
+  {
+    name: 'English (English, America)',
+    code: 'en-US',
+  },
+  {
+    name: '中文 (Chinese, S)',
+    code: 'zh-CN',
+  },
+  {
+    name: 'Czech (Czech)',
+    code: 'cs',
+  },
+  {
+    name: 'Croatian (Croatian, Croatia)',
+    code: 'hr-HR',
+  },
+  {
+    name: 'Dansk (Danish)',
+    code: 'da',
+  },
+  {
+    name: 'Deutsch (German, Germany)',
+    code: 'de-DE',
+  },
+  {
+    name: 'Dutch (Dutch, Netherlands)',
+    code: 'nl-NL',
+  },
+  {
+    name: 'Espa&ntilde;ol (Spanish)',
+    code: 'es',
+  },
+  {
+    name: 'Esperanto',
+    code: 'eo',
+  },
+  {
+    name: 'Français (French, Canada)',
+    code: 'fr-CA',
+  },
+  {
+    name: 'Français (French, France)',
+    code: 'fr',
+  },
+  {
+    name: 'Greek (Greek)',
+    code: 'el',
+  },
+  {
+    name: 'Italiano (Italian)',
+    code: 'it',
+  },
+  {
+    name: '日本語 (Japanese, Japan)',
+    code: 'ja-JP',
+  },
+  {
+    name: '한국어 (Korean)',
+    code: 'ko',
+  },
+  {
+    name: 'Polski (Polish)',
+    code: 'pl',
+  },
+  {
+    name: 'Português (Portuguese, Brazil)',
+    code: 'pt-BR',
+  },
+  {
+    name: 'Română (Romanian)',
+    code: 'ro',
+  },
+  {
+    name: 'Russian (Russian)',
+    code: 'ru',
+  },
+  {
+    name: 'Slovenský jazyk (Slovak)',
+    code: 'sk',
+  },
+  {
+    name: 'Turkish (Turkish)',
+    code: 'tr',
+  },
+  {
+    name: 'Українська (Ukrainian)',
+    code: 'uk',
+  },
+  {
+    name: 'Uzbek (Uzbek)',
+    code: 'uz',
+  },
+  {
+    name: 'Klingon (Klingon)',
+    code: 'tlh',
+  },
+];
```

Comment:
We should probably start with just 2 languages, English and a placeholder (like Spanish), and add more as translations are created.

We don't know, for example, if we'll get a Uzbek translation at all, since we have to wait for volunteers to do the translation.